bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBS/WGjj_ewk_qcd/WGjj_ewk_qcd_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+import model sm-ckm_no_b_mass
+define p = g u c d s b u~ c~ d~ s~ b~
+define j = p
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate  p p > l+ vl j j a QED=5 QCD=99 @ 1
+add process  p p > l- vl~ j j a QED=5 QCD=99 @2
```

Comment:
![image](https://user-images.githubusercontent.com/17442656/152040640-267e48f7-fe17-4dd8-bb08-2a9b10bce2a1.png)

Isn't this prone to this process (swap Z with gamma) without $$ t t~? @agrohsje Not sure how QED=X and QCD=Y will act here though.

Reply:
I just checked several others,
The naming conventions here is not so systematic which confuses me a bit. Some has "notop" and gives $$ t t~ but some doesn't have "notop" but gives $$ t t~. Some has "G" for photons, some has "A"

Reply:
You mean with t > wb > l nu b as a subprocess of the more generic lnuj ? Good point. I remember that we discussed this in one of the GEN meetings. Would that be vetoed as signal at the analysis level?

Reply:
@agrohsje Yup that was my concern here.

@andrea21z Can you comment on vetoing the top stuffs.
The technical settings are OK, but due to various naming conventions, I cannot really identify which physics process you aim for. If you are confident with this, I am fine to merge.

Reply:
> I just checked several others, The naming conventions here is not so systematic which confuses me a bit. Some has "notop" and gives $$ t t~ but some doesn't have "notop" but gives $$ t t~. Some has "G" for photons, some has "A"

Honestly, I cloned the existing requests leaving the same name of the dataset, but I can change it so that everything is consistent.

Reply:
> ![image](https://user-images.githubusercontent.com/17442656/152040640-267e48f7-fe17-4dd8-bb08-2a9b10bce2a1.png)
> 
> Isn't this prone to this process (swap Z with gamma) without $$ t t~? @agrohsje Not sure how QED=X and QCD=Y will act here though.

The analysis that requested these samples is looking for VBF, so (if I've understood well) should I put  '$$ t t~' in all of them?

Reply:
if that's the case, that is what I would do.